experimental/msal-react/src/index.ts

```diff
@@ -0,0 +1,22 @@
+// TODO: How do we get the token before executing an API call in pure TypeScript, outside of a React component context?
+// TODO: How do we raise the `error` state into the MsalContext, where changes will allow all subscribed components to update?
```

Comment:
If I'm understanding correct, event API should address this?

Reply:
This is exactly right. Today we have to wrap the MSAL API to intercept values and add to the Context Provider. When the event API goes live a wrapper won't be necessary, so the consumer just needs to call the MSAL functions as usual.

I also realized that while the event API isn't necessarily a blocker to continue building out the Alpha React MSAL library. I've started experimenting with creating a temp wrapper around MSAL that raises events when interacting with the MSAL API. For now this works, and so long as we agree early on a contract for which events MSAL will dispatch, then the demo can be built out accordingly. When the Event API goes live we just need to remove the temp wrapper and access MSAL API directly.

This was the approach I had been planning to use short-term for our apps in production.